rest/tests/test_search.py

```diff
@@ -181,16 +181,19 @@ def test_sortby(self):
         resource1.xa_avg_grade = 2.0
         resource1.xa_nr_attempts = 4
         resource1.xa_nr_views = 1000
+        resource1.title = '00000'
         resource1.save()
 
         resource2.xa_avg_grade = 4.0
         resource2.xa_nr_attempts = 1
         resource2.xa_nr_views = 100
+        resource2.title = '11111'
```

Comment:
Can you swap the titles such that they aren't in the same order as the resource id? That way we can confirm that it's the title being sorted and not the resource id